.vscode/launch.json

```diff
@@ -7,9 +7,15 @@
             "type": "extensionHost",
             "request": "launch",
             "runtimeExecutable": "${execPath}",
-            "args": ["--extensionDevelopmentPath=${workspaceRoot}"],
+            "args": [
```

Comment:
Revert this file for the PR

Reply:
this should be for my local validation, got merged accidentally!